pkg/config/editor_presets.go

```diff
@@ -48,30 +58,35 @@ func getPreset(osConfig *OSConfig, guessDefaultEditor func() string) *editPreset
 			editTemplate:              "hx -- {{filename}}",
 			editAtLineTemplate:        "hx -- {{filename}}:{{line}}",
 			editAtLineAndWaitTemplate: "hx -- {{filename}}:{{line}}",
+			openDirInEditorTemplate:   "hx {{dir}}",
```

Comment:
For consistency with the other templates, this should be
```suggestion
			openDirInEditorTemplate:   "hx -- {{dir}}",
```
Also for the other editors below, of course.

It's just a minor robustness thing, probably not relevant in practice. But it makes the command work even if somebody calls their worktree folder `--wait`. 😄

Reply:
Good pickup haha. It is interesting that now pretty much every editor uses the same command for opening a file as they do for opening a directory. The only exception is vscode which opens a file in the same window but opens a directory in a separate window. I'm a vscode guy so I don't know what the typical user of a terminal editor would want, or whether the behaviour is even configurable.